x/ccv/child/module.go

```diff
@@ -249,6 +278,32 @@ func (am AppModule) OnChanOpenInit(
 		return err
 	}
 
+	///////////////////////////////////////////////////
+	// Initialize distribution token transfer channel
+	// NOTE the handshake for this channel is handled by
+	//      the ibc-go/transfer module
+
+	transferPortID := "transfer"
```

Comment:
nit: Use the constant available in transfer/types in ibc-go

Reply:
updated (removed this variable altogether)